src/java/org/apache/cassandra/index/sai/disk/format/Version.java

```diff
@@ -41,16 +42,18 @@ public class Version
     public static final Version BA = new Version("ba", V2OnDiskFormat.instance, (c, i) -> stargazerFileNameFormat(c, i, "ba"));
     // Converged Cassandra with JVector
     public static final Version CA = new Version("ca", V3OnDiskFormat.instance, (c, i) -> stargazerFileNameFormat(c, i, "ca"));
+    // Encode trie entries using their AbstractType to ensure trie entries are sorted for range queries and are prefix free.
+    public static final Version DA = new Version("da", V4OnDiskFormat.instance, (c, i) -> stargazerFileNameFormat(c, i, "da"));
 
     // These are in reverse-chronological order so that the latest version is first. Version matching tests
     // are more likely to match the latest version so we want to test that one first.
-    public static final List<Version> ALL = Lists.newArrayList(CA, BA, AA);
+    public static final List<Version> ALL = Lists.newArrayList(DA, CA, BA, AA);
```

Comment:
I feel like we should move to a new version naming scheme that does not conflict with the version scheme ASF C* uses.

Reply:
I'll use `DB`.